webgui/index.html

```diff
@@ -22,7 +22,7 @@
 
     <!-- Third party libraries -->
 
-    <script type="text/javascript" src="scripts/node_modules/thrift/lib/js/src/thrift.js"></script>
+    <script type="text/javascript" src="scripts/node_modules/thrift/lib/nodejs/lib/thrift/thrift.js"></script>
```

Comment:
This might be a correct path for Thrift version 0.12.0, but for the most recent version 0.13.0, the correct path would be `scripts/node_modules/thrift/src/thrift.js`. I guess the Thrift version cannot be simply `>=0.12.0`, but should be `0.12.0` precisely.